notebook/06_xgboost_trade.ipynb

```diff
@@ -0,0 +1,720 @@
S6Gvr49+/fph4MCBGDBgAHr37o2ePXvCyMhI7shEjSI3NxexsbGIiYmRPv/x8fFQKpV47LHHMHDgQGkbOnQorKys5I5M/08IARsbG6xcuRJLly6VO06jq62tRUxMDM6dO4eLFy8iPDwcV69ehVKphK2tLTw8PODh4YEBAwbAzc2NBUdqlWpqapCUlCS1weHh4YiMjERxcTH09fXRv39/eHh4YNCgQRg2bBg/563A+fPnMXfuXCgUCnz00Uf45z//KXckIiIiIqLmwkI71adUKnH16lWp8BgREYHLly+joqICGhoacHZ2hqurK9zc3ODm5gZXV1f07NkThoaGckcnuqu8vDxcuXIFcXFxiI2NRVxcHK5cuYJbt24BAMzNzaVvcbi7u2PgwIEs6LRw165dQ5cuXXDmzBkMHTpU7jjNoqSkBJGRkQgPD0d4eDguXryItLQ0AICpqSnc3NzQq1cvtc3a2lre0ET4b8dYWloaYmNjceXKFcTExODKlSu4evUqqqqqoKWlhZ49e2LQoEFSYb13795cS6aVKi8vx9q1a/Hxxx9j/Pjx2LFjBzp16iR3LCIiIiKipsZCOz0YpVKJ1NRUxMTE4OrVq1KxMj4+HlVVVdDU1ISzszNcXFzQrVs3dO3aFV27dkW3bt3QpUsXTkFDTa6wsBApKSlISUlBcnKy9Dg+Ph4KhQLAf4uRqk4iV1dX9OrVC66uriwAtEL79+/HtGnTUFhYCGNjY7njyKagoKBe8TImJga5ubkAACsrK7i6uqJbt27Spmqn+e0kamy5ublISkpCcnKy2n/j4+NRXFwMAHByclLrFFK1x/r6+jKnp8Z26tQpvPDCC8jLy8PWrVsxe/ZsuSMRERERETUlFtrp0dTU1CAlJQWxsbG4evWq9A/rlJQUabSwpqYmOnbsqFaAd3R0hKOjIzp16gR7e3vo6urKfCXU0pWVleHGjRu4efMmMjIykJqaKhXTU1JSkJOTAwDQ0tKCg4OD9FlzcXGRpj1ydHSU+SqosbzzzjvYu3cv4uPj5Y7SIikUCmlapISEBCQnJyM5ORnp6emora0FANjZ2UlF927dusHJyUlqm+3t7aGtrS3zVVBLU1FRgRs3biA9PR03b96UOjZVW35+PgBAV1dX6nx3cXFBjx49pKI6F41uX0pLS7Fy5Ups27YNU6dOxZdffgkLCwu5YxERERERNQUW2qnplJSUqBVCVf8gT01Nxc2bN1FdXQ0A0NDQgK2tLRwcHNCxY0c4ODioPba2toa9vX27HrXa1uXk5CAnJwcKhQI3b96UCuqqxxkZGcjLy5Neb2BggM6dO0vfmlAV1bt27YrOnTuz46Yd8PHxgYGBAfbs2SN3lFalsrISqampUqdo3a1uu6ylpQVbW1s4OTnBwcEBnTp1UivC29vbw8rKit9WakNKS0uRlZWFW7duIT09XSqmX79+XfpZ1aEJ/K8drttZo9ocHR2hpaUl49VQS3PixAk8//zzUCqVCAgIwIQJE+SORERERETU2FhoJ3nU1tZCoVAgPT0dGRkZSE9PlwqqqscKhQI1NTXSewwMDGBtbQ07OztYWVmpPbaxsYGdnR06dOgACwsLWFhYcM54GRUVFSE/Px/5+fnIycnBrVu3kJ2dDYVCgVu3biEnJwdZWVnIzs5GTk6OVNwDAH19fbWOFkdHR7UOmE6dOsHS0lLGq6OWwMnJCf/617+watUquaO0Gap2WVVYVRVZVR1f6enp0jeVVDp06AAbGxupQ9Ta2lpqj62trWFtbQ1LS0tYWFjA3Nxcpitrn5RKJfLz85GXl4e8vDypDVYoFMjOzkZmZiays7Nx69YtZGVlobS0VHqvtrY27Ozs1L595uDgIHW8ODg44LHHHpPx6qg1KigowMKFC7Fnzx68/PLL+Oijj9hZR0RERERtCQvt1HIplUqpIJCVlSUVbBUKRb3H2dnZ0nQIKnp6erCwsFArvqs21T5jY2OYmJjA1NQUhoaGMDQ0lIr0hoaGMDU1lenq5SGEQEFBAUpLS1FWVobi4mIUFRWhrKwMZWVlyM/PR0lJiVS8URXTVZtqn1KpVDuurq6uVICztbWFlZUVbG1tYWNjo/ZY9TPR/eTn58PS0hKHDx/GxIkT5Y7TrlRUVCAzM1OtbVZ1pGVmZkrfTFEoFCgrK1N7r6amptT2dujQ4a6PVe2xqakpjIyMYGhoCHNzcxgbG8PIyKhdzStfWFiI0tJSlJaWoqioCMXFxdLPBQUFUoemqpCen5+P3NxcaV9hYWG9Y5qZmUmdIHZ2dmqdJKq2WLVx6iBqKoGBgVi8eDEcHR3xww8/oE+fPnJHIiIiIiJqDCy0U9tQW1uLnJwcteLv3QrBd+4vKSmRFmi7lzuL8ABgYmICbW1t6OnpSSPnVc8ZGRlBV1cXOjo69aa7MTc3h4aGxl3PY2ZmBk1NzXr7q6qq1EYa1lVdXY2SkhK1fQUFBRBCoKysDJWVlaipqZGusbCwELW1taioqEB5ebn0XFFREUpLS1FeXn7fe6HqnLhXB8bd9j/22GOcj5Ua1cmTJzF27FhkZmbCzs5O7jh0DyUlJcjOzlYrBP/d45KSEhQVFd33uKrOUCMjI5iYmEBDQ0MaLa9qm/X19WFgYAAtLS2pw/TONlbVVt+N6jh3UrWdd1NeXo6KigrpZ1XbrerABIDi4mLU1NRIx1EqldL15ufnq3Vy3ovqek1NTdXaXFWHxZ2dGBYWFrC0tISVlRUXHKUWIy0tDX5+foiIiIC/vz+WL19+178DERERERG1Iiy0EwGQRm2rRgrWLXYUFhZKI7pVxRJVwVpVWKmtrZVGD6oKKZWVlWojOusWvFU/K5VK6OnpqRVb7lS3iHQ3dxaxTU1NoaWlJRWaNDU1pcXnjI2NoaOjA11dXRgZGUlFKBMTE6lwdeeI/rodDUQtwebNm/Hhhx/Wm8aE2g5V519paSkKCwvrjeYuKSmRfr5bZ6Kqo7FuZ6RqoU4VVafkneoW
```

Comment:
https://github.com/yidong72/gQuant/blob/1f15ed0eee9540f2928653890f111587be3ff2e2/notebook/06_xgboost_trade.ipynb#L178-L180

Do you want to leave this as is? It's just that `graph_obj = load_workflow('../task_example/xgboost_trade.ymal')` is not a graph object. It's a list of dict entries. I think it's misleading to call it `graph_obj`.

We could introduce a workflow class that the load workflow function could return. This workflow object could have methods like viz_graph and draw. Then other functions could work with a workflow object instead of passing around a list of dictionaries.

Reply:
Agree with you. I will create a ticket to make the workflow class and make a separate PR for it